hetmech/test_matrix.py

```diff
@@ -0,0 +1,28 @@
+import hetio.readwrite
+from .matrix import metaedge_to_adjacency_matrix
+
+'''
+Test the functionality of metaedge_to_adjacency_matrix in
+generating sparse matrices vs numpy arrays. Uses same test
+data as in test_degree_weight.py Figure 2D of Himmelstein &
+Baranzini (2015) PLOS Comp Bio.
+https://doi.org/10.1371/journal.pcbi.1004259.g002
+'''
+
+url = 'https://github.com/dhimmel/hetio/raw/{}/{}'.format(
+    '9dc747b8fc4e23ef3437829ffde4d047f2e1bdde',
+    'test/data/disease-gene-example-graph.json',
+)
+graph = hetio.readwrite.read_graph(url)
```

Comment:
This should go inside of the test function (or in another function that the test calls). As is, this graph will be loaded whenever the module is imported.